frontend/packages/dev-console/src/components/topology/components/nodes/MonitoringDecorator.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens';
+import * as UIActions from '@console/internal/actions/ui';
+import { Node } from '@console/topology';
+import Decorator from './Decorator';
+
+interface MonitoringDecoratorProps {
+  radius: number;
+  x: number;
+  y: number;
+  element: Node;
+  onClickTab?: (name: string) => void;
+}
+
+const MonitoringDecorator: React.FC<MonitoringDecoratorProps> = ({
+  radius,
+  x,
+  y,
+  element,
+  onClickTab,
+}) => {
+  return (
+    <Tooltip key="monitoring" content="Monitoring" position={TooltipPosition.left}>
+      <Decorator
+        x={x}
+        y={y}
+        radius={radius}
+        onClick={(e) => {
+          e.stopPropagation();
+          onClickTab('Monitoring');
+          element
+            .getGraph()
+            .getController()
+            .fireEvent('selection', [element.getId()]);
```

Comment:
Interesting approach. It'll work :)

Use the const for the selection event instead of a string